docs/reference/release-notes/highlights.asciidoc

```diff
@@ -38,3 +38,186 @@ Other versions:
 // === Heading
 //
 // Description.
+
+// tag::notable-highlights[]
+[discrete]
+=== EQL
+
+EQL (Event Query Language) is a declarative language dedicated for identifying patterns and relationships between events.
+
+Consider using EQL if you:
+
+* Use {es} for threat hunting or other security use cases
+* Search time-series data or logs, such as network or system logs
+* Want an easy way to explore relationships between events
+
+A good intro on EQL and its purpose is available
+https://www.elastic.co/blog/introducing-event-query-language[in this blog
+post]. See the <<eql,EQL in Elasticsearch>> documentaton for an in-depth
+explanation, and also the
+https://eql.readthedocs.io/en/latest/query-guide/index.html[language
+reference]
+
+This release includes the following features:
+
+* Event queries
+* Sequences
+* Pipes
+
+An in-depth discussion of EQL in ES scope can be found at {es-issue}49581[#49581].
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[discrete]
+=== Data streams
+
+A _data stream_ is a convenient, scalable way to ingest, search, and manage
+continuously generated time-series data. The provide a simpler way to split
+data across multiple indices and still query it via a single named resource.
+
+See the <<data-streams,Data streams documentation>> to get started.
+// end::notable-highlights[]
+
+
+// tag::notable-highlights[]
+[discrete]
+=== Enable fully concurrent snapshot operations
+
+Snapshot operations can now execute in a fully concurrent manner.
+
+* Create and delete operations can be started in any order
+* Delete operations wait for snapshot finalization to finish, are batched
+  as much as possible to improve efficiency and, once enqueued in the
+  cluster state, prevent new snapshots from starting on data nodes until
+  executed
+* Snapshot creation is completely concurrent across shards, but per shard
+  snapshots are linearized for each repository, as are snapshot
+  finalizations
+
+// end::notable-highlights[]
+
+
+// tag::notable-highlights[]
+[float]
+=== Improve speed and memory usage of multi-bucket aggregations
+
+Before {minor-version}, many of our more complex aggregations made a simplifying
+assumption that required that they duplicate many data structures once per
+bucket that contained them. The most expensive of these weighed in at a
+couple of kilobytes each. So for an aggregation like:
+
+----
```

Comment:
The CI checks are failing here as follows:

> 04:24:16 FAILURE: Build failed with an exception.
04:24:16 
04:24:16 * What went wrong:
04:24:16 Execution failed for task ':docs:buildRestTests'.
04:24:16 > reference/release-notes/highlights.asciidoc[109:123]: Snippet missing a language.

Reply:
I added "[source,console]", hopefully that fixes CI.